lib/workers/pr/changelog/release-notes.ts

```diff
@@ -157,11 +187,18 @@ export async function getReleaseNotesMd(
   }
   let changelogFile: string;
   let changelogMd = '';
+  let apiTree: string;
+  let apiFiles: string;
   try {
-    let apiPrefix = apiBaseUrl.replace(/\/?$/, '/');
-
-    apiPrefix += `repos/${repository}/contents/`;
-    const filesRes = await http.getJson<{ name: string }[]>(apiPrefix);
+    const apiPrefix = apiBaseUrl.replace(/\/?$/, '/');
+    if (apiBaseUrl.includes('gitlab')) {
+      apiTree = apiPrefix + `projects/${repository}/repository/tree/`;
+      apiFiles = apiPrefix + `projects/${repository}/repository/files/`;
+    } else {
+      apiTree = apiPrefix + `repos/${repository}/contents/`;
+      apiFiles = apiTree;
+    }
+    const filesRes = await http.getJson<{ name: string }[]>(apiTree);
```

Comment:
Here we use GithubHttp, which would add github credentials but we maybe request gitlab api, we need to use `glGot` here.

So the request should be in the condition above.